content/tutorials/presence.textile

```diff
@@ -0,0 +1,306 @@
+
+A single @client ID@ may be present multiple times on the same channel, via different client connections; but each will be seen as a different member of the presence set, as they are differentiated by their unique @connection ID@. For example, if a client with ID “Sarah” is connected to a chat channel on both a desktop and a mobile device simultaneously, “Sarah” will be present twice in the presence member set with the same client ID, each with a different @connection ID@. A member of the presence set is therefore unique by the combination of the @client ID@ and @connection ID@ strings.
+
+h3. Step 3a - Updating your presence data
+
+In the previous step, we set a payload for our entry in the presence set: the string 'hello', which other clients will receive with the notification that we've entered. We can update that whenever we want, and other clients will receive the update:
+
+blang[javascript,nodejs].
+  ```[javascript]
+    channel.presence.update('howdy!', function(err) {
+      console.log('If no err, we have successfully updated our data');
+    })
+  ```
+
+  "See this step in Github":https://github.com/ably/tutorials/commit/#######################TODO
+
+blang[ruby].
+  ```[ruby]
+      channel.presence.update(data: 'hello') do
+        puts "We have successfully updated our data"
+      end
+    end
+  ```
+
+  "See this step in Github":https://github.com/ably/tutorials/commit/######################TODO
+
+h2. Step 4 - Get present members
+
+The next thing to do is to find out who else is present. (Spoiler: it's currently just you).
+
+blang[javascript,nodejs].
+  ```[javascript]
+    channel.presence.get(function(err, members) {
+      console.log('There are ' + members.length + ' clients present on this channel');
+      var first = members[0];
+      console.log('The first member is ' + first.clientId);
+      console.log('and their data is ' + first.data);
+    })
+  ```
+
+  "See this step in Github":https://github.com/ably/tutorials/commit/#######################TODO
+
+blang[ruby].
+  ```[ruby]
+      channel.presence.get do |members|
+        puts "There are #{members.length} clients present on this channel"
+        first = members[0]
+        puts "The first member is #{first.clientId} and their data is #{first.data}"
+      end
+    end
+  ```
+
+  "See this step in Github":https://github.com/ably/tutorials/commit/######################TODO
+
+As soon as you attached to the channel, it started to sync with Ably. When you make this call to get the presence set, it will (by default) wait until it has finished syncing before completing. For small presence sets this will take only a few milliseconds; for larger ones (with hundreds of members) the full sync can take a little longer.
```

Comment:
s/you attached to the channel, it started to sync/you are attached to the channel, the presence set starts to sync with Ably/g